packages/ui/src/types/authenticator/validator.ts

```diff
@@ -6,18 +7,27 @@ import { AuthChallengeNames } from './user';
  */
 export type ValidationError = Record<string, string>;
 
+export interface ValidationComplexityError {
+  password_complexity: Array<string>;
```

Comment:
I deleted this interface, and I renamed it to just `password`. I believe in other places in the code we have `confirm_password` but we do that because it matches the `name` of the field.